src/custom/hooks/useGetGpPriceStrategy.ts

```diff
@@ -0,0 +1,44 @@
+import ms from 'ms.macro'
+import { useState, useEffect } from 'react'
+import { DEFAULT_GP_PRICE_STRATEGY } from 'constants/index'
+
+export type GpPriceStrategy = 'COWSWAP' | 'LEGACY'
+// TODO: use actual API call
+// https://github.com/gnosis/gp-v2-contracts/issues/904
+export async function checkGpPriceStrategy(): Promise<GpPriceStrategy> {
+  return new Promise((accept) => setTimeout(() => accept(DEFAULT_GP_PRICE_STRATEGY), 500))
+}
```

Comment:
What's the progress on this?
I checked the mentioned issue but it doesn't contain any progress

Reply:
will ask backend - @nlordell have you guys been able to take a look at this? let me know if this is something i can help with

Reply:
I was under the impression we decided to add this to Github configuration to make it easier to change.

Reply:
Here a link to the Slack thread:
https://gnosisinc.slack.com/archives/CPZA1AGKY/p1637062028104200